tests/app/phase-tester-response/issue-responded/issue-responded.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { IssueRespondedComponent } from '../../../../src/app/phase-tester-response/issue-responded/issue-responded.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { IssueComment } from '../../../../src/app/core/models/comment.model';
+import { USER_JUNWEI } from '../../../constants/data.constants';
+
+describe('IssuesPendingComponent', () => {
+  describe('.ngOnInit()', () => {
+    const issuesRespondedComponent = new IssueRespondedComponent();
```

Comment:
I think that it would be better to follow what the previous authors have done so far, which is to only init the component before each test! For example: 

https://github.com/CATcher-org/CATcher/blob/7ab1caba68677c9e9a093f706cc34ec4d2924411/tests/app/phase-team-response/issues-pending/issues-pending.component.spec.ts#L23-L28

Reply:
Hmm I was thinking that it was somewhat redundant since the filter function is static and stateless.

Are there any distinctive differences between the 2 ways?

Reply:
Actually, coming to think of it. you way is actually better and what you mentioned, the filter function is stateless 😅

Reply:
I will resolve this conversation!